src/seedsigner/helpers/mnemonic_generation.py

```diff
@@ -121,3 +131,17 @@ def generate_mnemonic_from_image(image, wordlist_language_code: str = SettingsCo
 
     # Return as a list
     return bip39.mnemonic_from_bytes(hash.digest(), wordlist=Seed.get_wordlist(wordlist_language_code)).split()
+
+
+def generate_checksum_from_entropy(entropy_bits: str) -> str:
```

Comment:
This is unnecessary. `embit`'s `mnemonic_from_bytes` will calculate it for us.

see my proposed implementation: https://gist.github.com/kdmukai/1a666c945c46ada120fd2cd3c5a1f33c